script.js

```diff
@@ -0,0 +1,227 @@
+                   <li><button class="projects-btn" type="button">Bootstrap</button></li>
+                   <li><button class="projects-btn" type="button">Ruby</button></li>
+               </ul>
+           </div>
+   </div>  
+         
+        </div>
+        <button data-modal-target="#modal" class="see-project">See project</button>
+   </div>
+   <div class="see-proj">
+    <div class="projects-background" id="bg4"> 
+       <div class="background">
+        <h2 class="projects-heading">Professional Art <br>printing Data</h2>
+        <h2 class="Desktop-heading">Profesional Art Printing Data More</h2>
+       <p class="para">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.</p>
+       <div class="project-btn">
+           <ul>
+               <li><button class="projects-btn" type="button">HTML</button></li>
+               <li><button class="projects-btn" type="button">Bootstrap</button></li>
+               <li><button class="projects-btn" type="button">Ruby</button></li>
+           </ul>
+       </div>
+</div>  
+     
+    </div>
+    <button data-modal-target="#modal" class="see-project">See project</button>
+</div>
+<div class="see-proj">
+    <div class="projects-background" id="bg5" > 
+       <div class="background">
+        <h2 class="projects-heading">Professional Art <br> printing Data</h2>
+        <h2 class="Desktop-heading">Data Dashboard Healthcare</h2>
+       <p class="para">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.</p>
+       <div class="project-btn">
+           <ul>
+               <li><button class="projects-btn" type="button">HTML</button></li>
+               <li><button class="projects-btn" type="button">Bootstrap</button></li>
+               <li><button class="projects-btn" type="button">Ruby</button></li>
+           </ul>
+       </div>
+</div>  
+     
+    </div>
+    <button data-modal-target="#modal" class="see-project">See project</button>
+</div>
+
+
+<div class="see-proj">
+    <div class="projects-background" id="bg6"> 
+       <div class="background">
+        <h2 class="projects-heading">Professional Art <br> printing Data</h2>
+            <h2 class="heading3" >website Portfolio</h2>
+       <p class="para">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required. Has been the <br> industry's standard.</p>
+       <div class="project-btn">
+           <ul>
+               <li><button class="projects-btn" type="button">HTML</button></li>
+               <li><button class="projects-btn" type="button">Bootstrap</button></li>
+               <li><button class="projects-btn" type="button">Ruby</button></li>
+           </ul>
+       </div>
+    </div> 
+    </div>
+</div>
+  <button data-modal-target="#modal" class="see-project">See project</button>                    
+    </div>       
```

Comment:
- [Optional] KIndly, the main requirements of this milestone are: 👇
1. to create all the projects cards dynamically
2. Then, build popups for each card dynamically also.
To fulfill these both requirements, we make use of the information we stored in the array of objects in the JS file. Right now, you are not creating the project cards dynamically so please consider creating all the cards dynamically from the same array of objects instead creating the whole cards with string literals.
This way adding projects in the future will be easy, all you need to do would be to add another object in that array.
After that, implementing the popup window dynamically will be easy. 👍